performance/cinterop/gradle.properties

```diff
@@ -0,0 +1 @@
+org.jetbrains.kotlin.native.home=../../dist
```

Comment:
Is it indeed needed?

Reply:
Before it was needed, because it was used from subproject directory. I can test later again when this functionality will be fixed.

Reply:
Yes, it's needed
```
 Classpath of the tool is empty: cinterop
  Probably the 'org.jetbrains.kotlin.native.home' project property contains an incorrect path.```